app/models/user.rb

```diff
@@ -37,4 +41,20 @@ def find_badge_assignment(badge)
   def admin_module?(current_module)
     roles.pluck(:name).include? "admin_#{current_module}"
   end
+
+  private
```

Comment:
Useless `private` access modifier.<br>Keep a blank line before and after `private`.